lib/plugins/helper/meta_generator.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-function metaGeneratorHelper() {
-  return `<meta name="generator" content="Hexo ${this.env.version}">`;
-}
-
-module.exports = metaGeneratorHelper;
+module.exports = ctx => {
+  return () => {
+    if (!ctx.extend.helper.getProp('meta_generator')) ctx.extend.helper.setProp('meta_generator', true);
+    return `<meta name="generator" content="Hexo ${ctx.env.version}">`;
```

Comment:
Since this helper is [bind](https://github.com/hexojs/hexo/pull/4070/files#diff-0c88d2f71e5c4ea3ed9b3c0933b15686R46) to ctx, I believe `ctx.version` should work?

Reply:
@curbengh I will have a test on a dummy site to see if it will cause `undefined` again.

Reply:
Update to `ctx.version`.

Reply:
also verified in my test.